appdatabase/migrations/sql/1674570455_communities_permissions.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS communities_permissions (
```

Comment:
I think community singular is more grammatically accurate and it sounds better
```suggestion
CREATE TABLE IF NOT EXISTS community_permissions (
```

Reply:
👍